src/main/java/com/google/sps/GetConfigProperties.java

```diff
@@ -32,11 +32,14 @@ public static String getPropertyValue(String filepath, String propertyName) thro
 
   /**
    * This function retrieves the value of the API Key from the app.properties file. 
-   * @return String                 This return the String value of the API key.
-   * @throws FileNotFoundException  This exception is thrown if the filepath given does not exist. 
-   * @throws IOException            This exception is thrown if the filepath and/or the propertyName do not exist. 
+   * @return String     This return the String value of the API key.
    */
-  public static String getApiKey() throws FileNotFoundException, IOException {
-    return getPropertyValue("app.properties", "api_key");
+  public static String getApiKey() {
+    try {
+      return getPropertyValue("app.properties", "api_key");
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+    return "";
```

Comment:
Could you please explain why you return ""? What happens when you build with an empty ApiKey?

Reply:
Nothing happens, it just returns null otherwise. I thought returning an empty String might better than returning null since the function expects to return a String, but I am not sure what is better (I was reading through that and did not really have a clear answer: https://softwareengineering.stackexchange.com/questions/120355/is-it-better-to-return-null-or-empty-values-from-functions-methods-where-the-ret)

If you prefer I return null, I can make the change